### PR TITLE
Fix inaccurate MongoDB Metrics & Add label for complex MQE in Nginx dashboard.

### LINKED ISSUE #11982: [Bug] Metrics are not aggregated properly as OTEL collector targeting per node
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

This issue was found during Clickhouse PR review, https://github.com/apache/skywalking/pull/11966#pullrequestreview-1912778007

We noticed this is a common bug case in previous monitoring. We should check them and update the status here.

- [x] MongoDB @weixiang1862 https://github.com/apache/skywalking/pull/12126
- [x] MySQL @yswdqz https://github.com/apache/skywalking/pull/12130
- [x] MariaDB @yswdqz https://github.com/apache/skywalking/pull/12130
- [x] PostgreSQL @yswdqz https://github.com/apache/skywalking/issues/11982
- [x] Redis @yswdqz https://github.com/apache/skywalking/pull/12130
- [x] Elasticsearch @innerpeacez 
- [x] Nginx @weixiang1862 https://github.com/apache/skywalking/pull/12096
- [x] APISIX @pg-yang https://github.com/apache/skywalking/pull/12108
- [x] Kafka @wankai123 
- [x] Pulsar @CodePrometheus  
- [x] RabbitMQ @innerpeacez https://github.com/apache/skywalking/pull/12144

### What you expected to happen

Service metrics should be labeled values, and only use MQE(query stage aggregation) to merge the data to build metrics accurately. 

In this case, if there are two nodes

In the case A
ck node1 -> OTEL -> OAP
ck node2-> OTEL -> OAP

In the case B
T1, ck node1 -> OTEL -> OAP
T2, ck node1 -> OTEL -> OAP
(T1 T2) are in the same minute(same for an hour/day)

Then from the OAP process perspective, it can't tell A and B, so, if you work on downsampling(default as AVG), it could be AVG of node1 and node2(case A), or AVG of ck node1 from T1 and T2(case B). 
For a more complex case, it could be a combination of case A and B, in one minute, N scrapings from node1 and M scrapings from node2, then the default sampling is doing AVG for N + M scrapings from node1 and node2. This actually is not the value of the aggregated metric for the whole service(multiple nodes).

### How to reproduce

Metrics from nodeA and nodeB will overlap with each other. Due to nodeA and nodeB will report data separately.

### Anything else

_No response_

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by weixiang1862**
### Fix inaccurate MongoDB Metrics
- [x] Add a unit test to verify that the fix works.
- [x] Explain briefly why the bug exists and how to fix it.

- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #11982 MongoDB part.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

1. change service `Document Avg QPS` to `Document Total QPS`
2. change service `Operation Avg QPS` to `Operation Total QPS`
3. change service `Document Avg QPS` to `Document Total QPS`
4. change service `Cursor Avg` to `Cursor Total`
5. change service `DB Avg Data Size Per Shard` to `DB Total Data Size`
6. change service `DB Avg Index Size Per Shard` to `DB Total Index Size`
7. change service `DB Avg Index Count Per Shard` to `DB Total Index Count`
8. change service `DB Avg Collection Count Per Shard` to `DB Total Collection Count`
9. filter [ARBITER node](https://www.mongodb.com/docs/manual/core/replica-set-arbiter/) when calculate repl_lag (ARBITER `mongodb_mongod_replset_member_replication_lag` metric value has no meaning and will effect the final result)
10. calculate node latency by two rate divided in MQE and remove it from MAL.

image

image

### CODE REVIEW

**Comment by wankai123** [comment]
I suggest adding the label to simplify the name of the metric, otherwise it will show the total expression if it's not a labeled metric.
And I found the Nginx dashboards have the same issue.
image

**Comment by weixiang1862** [comment]
> I suggest adding the label to simplify the name of the metric, otherwise it will show the total expression if it's not a labeled metric.
> And I found the Nginx dashboards have the same issue.
> image
> 

OK, I will check them.

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/mongodb/mongodb-cluster.json [hunk 1]
```diff
@@ -231,7 +232,7 @@
                     "tableHeaderCol2": ""
                   },
                   "expressions": [
-                    "meter_mongodb_cluster_db_data_size/1024/1024/1024"
+                    "aggregate_labels(meter_mongodb_cluster_db_data_size, sum(database))/1024/1024/1024"
```
Comment (wankai123) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/mongodb/mongodb-cluster.json:235]: suggest using the `latest` or `avg`... function to aggregate values if the widget can't show the time series values.

Comment (weixiang1862) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/mongodb/mongodb-cluster.json]: Updated.

**Comment by weixiang1862** [comment]
> I suggest adding the label to simplify the name of the metric, otherwise it will show the total expression if it's not a labeled metric. And I found the Nginx dashboards have the same issue. image

Should I fix nginx dashboard label in this pr too?

**Comment by wu-sheng** [comment]
Yes, please.

**Review by wankai123 — APPROVED** [review]

**Pull request merged**